sections/06_discussion.md

```diff
@@ -6,22 +6,10 @@ can be further pruned.*
+
+What is the state of the TF binding models based on these metrics? [Alipanahi et al. developed the DeepBind model](http://www.nature.com/nbt/journal/v33/n8/full/nbt.3300.html) with state of the art performance on the DREAM5 TF binding challenge. DeepBind is evaluated based on auROC of classifying a positive set of 500 ChIP-seq peaks and a negative set with 500 shuffled versions of the sequences in those ChIP-seq peaks that preserve dinucleotide frequencies. Follow-ups to DeepBind such as [Deep Motif by Lachantin et al.](https://arxiv.org/abs/1605.01133) and [Genetic Architect by Deming et al.](https://arxiv.org/abs/1605.07156), improve the auROC on DeepBind’s formulation to 0.95. [DeepSEA by Zhou et al.](http://www.nature.com/nmeth/journal/v12/n10/full/nmeth.3547.html#methods) and follow-ups such as [DanQ by Quang et al.](https://academic.oup.com/nar/article/44/11/e107/2468300/DanQ-a-hybrid-convolutional-and-recurrent-deep) report state of the art auROC and auPRC of classifying 17% of the genome with at least one TF binding site. [Qin et al.](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005403) developed a deep learning model that outperforms both DeepBind and DeepSEA on their respective tasks and evaluations. However, a chromosome-wide evaluation of this model reveals that its recall at 50% FDR is less than 10% for most TFs. This suggests that recall at 10% FDR would be 0% or close to 0% for most TFs. These evaluations reveal that auROC and auPRC metrics are not good indicators of the performance needed for targeted validation. In fact, it is not unusual to obtain chromosome-wide auROC > 0.99 for TF binding predictions but a recall of 0% at 10% FDR. Thus, it is important to evaluate models of TF binding and other biochemical activities such as histone modifications and chromatin accessibility using a recall at fixed FDR thresholds on complete held out chromosomes.
```

Comment:
I took out the discussion of TF binding results in 46dccc0 and tried to use it as a motivating example only